src/types/TimeSBVR.ts

```diff
@@ -0,0 +1,29 @@
+import * as TypeUtils from '../TypeUtils'
+
+export class TimeSBVR implements SBVRType<string, Date> {
+	types = {
+		postgres: 'TIME',
+		mysql: 'TIME',
+		websql: 'TEXT',
+		odata: {
+			name: 'Edm.DateTime'
+		},
+	}
+
+	fetchProcessing = (data:string, callback:Callback<Date>) => {
+		// We append the date of the epoch so that we can parse this as a valid date.
+		let date = new Date('Thu, 01 Jan 1970 ' + data)
+		callback(null, date)
+
+	}
+
+	validate = (value:any, required:boolean, callback:Callback<string>) => {
+		TypeUtils.validate.date(value, required, (err, value) => {
```

Comment:
`value` is a shadowed variable, can you rename it to `response` or something similar?